platformio.ini

```diff
@@ -8,7 +8,7 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-[env:nanoatmega328new]
+[env:nanoatmega328]
 platform = atmelavr
-board = nanoatmega328new
+board = nanoatmega328
```

Comment:
I have an "old bootloader" Nano on my PCB

Reply:
No problem with that, the config I was using have the benefit that is fully compatible with the Uno (yes, the nano "new" formware is just a uno in the size of a nano..)